src/ModelContextProtocol.AspNetCore/ModelContextProtocol.AspNetCore.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <IsPackable>true</IsPackable>
+    <PackageId>ModelContextProtocol.AspNetCore</PackageId>
```

Comment:
Tagging @mikekistler @jeffhandley for feedback on the choice of naming for the new package.